src/main/java/org/primefaces/extensions/component/inputphone/Country.java

```diff
@@ -0,0 +1,46 @@
+/**
+ * Copyright 2011-2018 PrimeFaces Extensions
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.primefaces.extensions.component.inputphone;
+
+/**
+ *
+ * @author Jasper de Vries &lt;jepsar@gmail.com&gt;
+ */
+public class Country {
+
```

Comment:
Best to make this implement Serializable and add an equals(), toString(), and hashCode(). Whenevre you have these POJO's you never know how someone might use them or put them in SessionScoped beans etc.